### ISSUE TITLE
iOS version

### ISSUE THREAD

**Issue by pdaddyo** [pdaddyo/soundbounce#102]
##

**Comment by pdaddyo** [comment]
First commit (no audio, just a web view to login screen - skeleton code): #173 - thanks Andrew for making a start here.

**Comment by horaciohikma** [comment]
Thanks pdaddyo/Andrew for the interest in having a IOS platform soon on SoundBounce. I know that is a hard work and to have an approval from Apple it should take some weeks/months. Please keep me informed regarding the follow up of this process. Thank you.

Regards.

**Comment by sanford78** [comment]
I would love to see an iOS version!!

**Comment by horaciohikma** [comment]
Yeah, it will be great but it seems that any develloper wants to. I'm sure that if it was available in IOS more users will be in soudbounce at work too

**Comment by horaciohikma** [comment]
No dia domingo, 20 de setembro de 2015, sanford78 <notifications@github.com>
escreveu:
